action.yaml

```diff
@@ -183,16 +225,6 @@ runs:
         # Run 'trunk check' on pull request
         ${GITHUB_ACTION_PATH}/pull_request.sh
       env:
-        GITHUB_EVENT_PULL_REQUEST_BASE_SHA:
-          ${{ env.GITHUB_EVENT_PULL_REQUEST_BASE_SHA || github.event.pull_request.base.sha }}
-        GITHUB_EVENT_PULL_REQUEST_HEAD_REPO_FORK:
-          ${{ env.GITHUB_EVENT_PULL_REQUEST_HEAD_REPO_FORK ||
-          github.event.pull_request.head.repo.fork }}
-        GITHUB_EVENT_PULL_REQUEST_HEAD_SHA:
-          ${{ env.GITHUB_EVENT_PULL_REQUEST_HEAD_SHA || github.event.pull_request.head.sha }}
-        GITHUB_EVENT_PULL_REQUEST_NUMBER:
-          ${{ env.GITHUB_EVENT_PULL_REQUEST_NUMBER || github.event.pull_request.number }}
-        INPUT_CHECK_RUN_ID: ${{ inputs.check-run-id }}
         INPUT_SAVE_ANNOTATIONS: ${{ inputs.save-annotations }}
```

Comment:
Are some things not included in the json? (save-annotations?)

Reply:
annotation saving/posting goes away with CheckService, because we no longer have to deal with the permission restriction that comes with the github workflow